### PR TITLE
Add new files from `flutter create .`

### PR DESCRIPTION
**Pull request by clocksmith**
This includes manifest.json which helps with PWA compatibility.

### CODE REVIEW

File: web/manifest.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "gallery",
+    "short_name": "gallery",
+    "start_url": ".",
+    "display": "minimal-ui",
+    "background_color": "#0175C2",
+    "theme_color": "#0175C2",
+    "description": "A new Flutter project.",
```
Comment (rami-a) [inline web/manifest.json:8]: should we update some of these values?

File: web/manifest.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "orientation": "portrait-primary",
+    "prefer_related_applications": false,
+    "icons": [
+        {
+            "src": "icons/Icon-192.png",
```
Comment (rami-a) [inline web/manifest.json:13]: Not necessary for this PR, but we should update all of these new assets to be the proper ones

Comment (clocksmith) [reply web/manifest.json:13]: OK, what are the proper ones?

File: web/manifest.json (hunk 1 above)
Comment (clocksmith) [reply web/manifest.json]: Done

File: web/manifest.json (hunk 2 above)
Comment (guidezpl) [reply web/manifest.json:13]: Let me get you some images

File: web/manifest.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "gallery",
+    "short_name": "gallery",
```
Comment (guidezpl) [inline web/manifest.json:3]: nit: Flutter Gallery

File: web/manifest.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "start_url": ".",
+    "display": "minimal-ui",
+    "background_color": "#0175C2",
+    "theme_color": "#0175C2",
```
Comment (guidezpl) [inline web/manifest.json:7]: 451B6F

Comment (clocksmith) [reply web/manifest.json]: Done

File: web/manifest.json (hunk 3 above)
Comment (clocksmith) [reply web/manifest.json]: Done

**Pull request merged**